docs/fastdds/xtypes/language_binding.rst

```diff
@@ -0,0 +1,827 @@
+
+        .. literalinclude:: /../code/DynamicTypesIDLExamples.idl
+            :language: idl
+            :start-after: //!--IDL_SEQUENCES
+            :end-before: //!--
+
+    .. tab:: XML
+
+        .. literalinclude:: /../code/XMLTester.xml
+            :language: xml
+            :start-after: <!-->XML_SEQUENCES<-->
+            :end-before: <!--><-->
+
+    .. tab:: C++
+
+        .. literalinclude:: /../code/DDSCodeTester.cpp
+            :language: c++
+            :start-after: //!--CPP_SEQUENCES
+            :end-before: //!--
+
+For a detailed explanation about the XML definition of this type,
+please refer to :ref:`Sequences <xmldynamictypes_sequence>`.
+
+.. _xtypes_supportedtypes_array:
+
+Arrays
+******
+
+Arrays are pretty similar to sequences with two main differences:
+They can have multiple dimensions and they do not need their elements to be stored consecutively.
+
+An array needs to know the number of dimensions it is managing.
+For that, users must provide a vector with as many elements as dimensions in the array.
+Each element in the vector represents the size of the given dimension.
+
+The |DynamicTypeBuilderFactory-api| exposes the function |DynamicTypeBuilderFactory::create_array_type| to 
+facilitate the creation of this type.
+
+The |DynamicData-api| class has a specific :func:`get__values` and :func:`set___values` functions for each 
+primitive type that allow users to work with multiple values at once.
+
+.. tabs::
+
+    .. tab:: IDL
+
+        .. literalinclude:: /../code/DynamicTypesIDLExamples.idl
+            :language: idl
+            :start-after: //!--IDL_ARRAYS
+            :end-before: //!--
+
+    .. tab:: XML
+
+        .. literalinclude:: /../code/XMLTester.xml
+            :language: xml
+            :start-after: <!-->XML_ARRAYS<-->
+            :end-before: <!--><-->
+
+    .. tab:: C++
+
+        .. literalinclude:: /../code/DDSCodeTester.cpp
+            :language: c++
+            :start-after: //!--CPP_ARRAYS
+            :end-before: //!--
+
+For a detailed explanation about the XML definition of this type,
+please refer to :ref:`Arrays <xmldynamictypes_array>`.
+
+.. _xtypes_supportedtypes_map:
+
+Maps
+****
+
+Maps contain a list of 'key-value' pair types, allowing users to insert, remove or modify the element types of the map.
+The main difference with sequences is that the map works with pairs of elements and creates copies of the key element 
+to block the access to these elements.
+
+The |DynamicTypeBuilderFactory-api| exposes the function |DynamicTypeBuilderFactory::create_map_type| to 
+facilitate the creation of this type.
+To create a map, users must set the types of the key and the value elements.
+Additionally, the size limit of the map. Users can use ``LENGTH_UNLIMITED`` to create unbounded sequences.
+
+The |DynamicData-api| class has a specific :func:`get__values` and :func:`set___values` functions for each primitive 
+type that allow users to work with multiple values at once.
```

Comment:
Instead of only talk about `get_value()` and `set_value()`, explain how the user access the key using `get_member_id_by_name` (explaining also each type has to be used here as `string`). Then with the MemberId retrieved, can modify the value.

Reply:
added in [e38d0d8](https://github.com/eProsima/Fast-DDS-docs/pull/730/commits/e38d0d856970dc4cdf6d7a04f3490344f7c903b1)